p2p/src/peer_manager/peerdb/mod.rs

```diff
@@ -162,14 +184,21 @@ impl<T: NetworkingService> PeerDb<T> {
 
     /// Checks if the given address is banned.
     pub fn is_address_banned(&mut self, address: &T::BannableAddress) -> bool {
-        if let Some(banned_till) = self.banned.get(address) {
+        if let Some(banned_till) = self.banned_addresses.get(address) {
             // Check if the ban has expired.
             let now = SystemTime::now()
```

Comment:
I just noticed this now. We never do that. We never pass time directly from the system. Otherwise we can have huge conflicts in tests.

Whether in this PR or another one, please use a `TimeGetter` object passed from the P2P object to know what time it's.

Reply:
OK, will do. But let's please do this in different PR.